pkg/detectlicense/licenses.go

```diff
@@ -21,20 +21,25 @@ type License struct {
 var (
 	Proprietary = License{Name: "proprietary"}
 
-	PublicDomain = License{Name: "public domain"}
-
-	Apache2 = License{Name: "Apache License 2.0", NoticeFile: true, Url: "https://opensource.org/licenses/Apache-2.0"}
-	BSD1    = License{Name: "1-clause BSD license", Url: "https://opensource.org/licenses/BSD-1-Clause"}
-	BSD2    = License{Name: "2-clause BSD license", Url: "https://opensource.org/licenses/BSD-2-Clause"}
-	BSD3    = License{Name: "3-clause BSD license", Url: "https://opensource.org/licenses/BSD-3-Clause"}
-	ISC     = License{Name: "ISC license", Url: "https://opensource.org/licenses/ISC"}
-	MIT     = License{Name: "MIT license", Url: "https://opensource.org/licenses/MIT"}
-	MPL2    = License{Name: "Mozilla Public License 2.0", NoticeFile: true,
-		WeakCopyleft: true, Url: "https://opensource.org/licenses/MPL-2.0"}
+	PublicDomain = License{Name: "Public domain"}
+
+	Apache2  = License{Name: "Apache License 2.0", NoticeFile: true, Url: "https://opensource.org/licenses/Apache-2.0"}
```

Comment:
I know I missed the boat on catching this in #6, so I'll point it out now: That should be `URL`, not `Url`

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Thanks for the link to Golang's code review guide. Knowing we're following this will help in future reviews.